app/bundles/PageBundle/Controller/Api/PageApiController.php

```diff
@@ -43,12 +55,4 @@ public function getEntitiesAction()
 
         return parent::getEntitiesAction();
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    protected function preSerializeEntity(&$entity, $action = 'view')
```

Comment:
Why was this removed? I worry about changing the behavior in this PR

Reply:
There is no `url` property in the entity